README.md

```diff
@@ -18,7 +18,8 @@ When setting up the **plugin**:
 * In the "Script source" tab:
   * Set "Source type" as "git";
   * Paste the link to this repository as "Repository";
-  * Set "Path" as `research-cloud-plugin.yml`
+  * Set "Path" as `research-cloud-plugin.yml`;
+  * Set "Tag" to target "branch", if needed;
```

Comment:
Good point to add this here! Maybe mentioning that this can be a tag as well:

```suggestion
  * Set "Tag" to target "branch" or "tag", if needed;
```